docker-wrappers/OmicsIntegrator1/metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "dockerName": "omics-integrator-1",
```

Comment:
What is the convention for adding hyphens or not?

Reply:
I believe this was an arbitrary choice, but I preserved it from `build-containers.yml`.